bundles/org.openhab.binding.infokeydinrail/src/main/feature/feature.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.infokeydinrail-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+
+	<feature name="openhab-binding-infokeydinrail" description="infokeyDinRail Binding" version="${project.version}">
+		<feature>openhab-runtime-base</feature>
+		<bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.infokeydinrail/${project.version}</bundle>
+	</feature>
+	<feature name="org.openhab.binding.infokeydinrail" description="openHAB Add-ons :: Bundles :: infokeydinrail Binding" version="3.3.0.SNAPSHOT">
+		<details>This project contains the official add-ons of openHAB</details>
+		<feature prerequisite="true" dependency="false">wrap</feature>
+		<bundle start-level="80">mvn:com.pi4j/pi4j-core/1.4</bundle>
+		<bundle start-level="80">mvn:javax.xml.bind/jaxb-api/2.3.1</bundle>
+		<bundle start-level="80">mvn:javax.activation/javax.activation-api/1.2.0</bundle>
+		<bundle start-level="80">wrap:mvn:com.pi4j/pi4j-gpio-extension/1.3</bundle>
+		<bundle start-level="80">wrap:mvn:org.lastnpe.eea/eea-all/2.2.1</bundle>
+	</feature>
```

Comment:
You don't need this one. Remove it.

Reply:
I removed it and tried to compile again and the result was the same!

Reply:
Remote `noEmbedDependency.profile`. Why did you add that? It's usually wrong.

Reply:
@thanast look at the MCP23017 binding to see what you should use in the feature.xml and pom.xml. You don't need to include gson in the pom.xml and also should not use commons io preferably.

Reply:
> @thanast look at the MCP23017 binding to see what you should use in the feature.xml and pom.xml. You don't need to include gson in the pom.xml and also should not use commons io preferably.

I uploaded this project just for your help. It is not in the phase that can be distributed or can be used. Also I would share the devices in the near future for anyone who will be interested. I have search the web also for issues with karaf and pi4j but nothing found. The intrested thing is that exit another binding called org.opehab.binding.mcp23017 that uses exactly the same libraries do compilation completely to my pc. I tried to copy the project rename it and recompile it compiles successfully. I cannot understand what I missing...

Reply:
> > @thanast look at the MCP23017 binding to see what you should use in the feature.xml and pom.xml. You don't need to include gson in the pom.xml and also should not use commons io preferably.
> 
> I uploaded this project just for your help. It is not in the phase that can be distributed or can be used. Also I would share the devices in the near future for anyone who will be interested. I have search the web also for issues with karaf and pi4j but nothing found. The intrested thing is that exit another binding called org.opehab.binding.mcp23017 that uses exactly the same libraries do compilation completely to my pc. I tried to copy the project rename it and recompile it compiles successfully. I cannot understand what I missing...

Thank you Hilbrand greetings to The Netherlands from my wife who is Dutch!